.azure-pipelines/azure-pipelines-build.yml

```diff
@@ -48,6 +48,7 @@ jobs:
             docker_syncd_rpc_image: yes
             platform_rpc: bfn
             swi_image: yes
+            BUILD_OPTIONS: $(BUILD_OPTIONS) TELEMETRY_WRITABLE=y
```

Comment:
Do you mean as below?
BUILD_OPTIONS: ${{ parameters.buildOptions }} TELEMETRY_WRITABLE=y

Reply:
I see, thanks, I\`ll fix this. So it means that my change wouldn\`t  work at all?